lib/cli/ui/spinner/spin_group.rb

```diff
@@ -197,7 +197,13 @@ def wait
             sleep(PERIOD)
           end
 
-          debrief if @auto_debrief
```

Comment:
This feels like we should be doing
```ruby
@m.synchronize do
  debrief if @auto_debrief
  @tasks.all?(&:success)
end
```
with `debrief` reduced to just the `@tasks.each` call with block.

Reply:
`debrief` is public and could be called manually